### ISSUE TITLE
intellijIdea 2022.1.3 UI dialogs doesn't react on Enter button 

### ISSUE THREAD

**Issue by nikitasius** [aws/aws-toolkit-jetbrains#3193]
**Describe the bug**
You do java file copy-paste, or you add a new watch into DEBUG, the UI in IDE doesn't react when you press ENTER.
ctrl-ENTER works (advices by jetbrains support.

**To reproduce**
1) install your plugin 1.45-221
2) open java project
3) copypaste code file, rename (in ui dialog), press ENTER
4) nothing happens

**Expected behavior**
It close the UI dialog.

**Screenshots**
just UI window stucked and waiting you press the OK. Or "new watch" blinking and waiting for Enter.

**Your Environment**
```
IntelliJ IDEA 2022.1.3 (Ultimate Edition)
Build #IU-221.5921.22, built on June 21, 2022
Licensed to ME_HERE
Subscription is active until July 19, 2022.
Evaluation purpose only.
Runtime version: 11.0.15+10-b2043.56 amd64
VM: OpenJDK 64-Bit Server VM by JetBrains s.r.o.
Linux 5.10.0-15-amd64
GC: G1 Young Generation, G1 Old Generation
Memory: 3072M
Cores: 12
Registry:
    ide.balloon.shadow.size=0

Non-Bundled Plugins:
    net.seesharpsoft.intellij.plugins.csv (2.18.2)
    com.perl5 (2022.1.1)
    com.mroche.JsonHelper (1.1.2)
    org.jetbrains.kotlin (221-1.7.0-release-281-IJ5591.52)
    org.asciidoctor.intellij.asciidoc (0.37.26)
    String Manipulation (9.5.0)
    Pythonid (221.5921.27)
    aws.toolkit (1.45-221)

Kotlin: 221-1.7.0-release-281-IJ5591.52
Current Desktop: XFCE
```

**Additional context**
Support told it's your plugin's issue. I confirm that in previous IDE worked fine. I updated Ultimate and got this. My Community worked fine till i updated it also.
Probably you or jetbrains broken something.

**Comment by cowwoc** [comment]
FYI, this also breaks the Watch window. Any time you type a new watch, you can no longer hit ENTER to save it.

**Comment by rli** [comment]
Fixed in `1.46`
